frontend/store/vaults.ts

```diff
@@ -0,0 +1,74 @@
+import { Vault, VaultTransaction } from 'auctions-core/src/types';
+import Vue from 'vue';
+import { fetchVault, fetchVaultsCount, liquidateVault } from 'auctions-core/src/vaults';
+import { ActionContext } from 'vuex';
+
+interface State {
+    vaultTransactions: Record<Vault['id'], VaultTransaction>;
+    vaultsCount?: number;
```

Comment:
Why do you need `vaultsCount` on the frontend? Is it displayed somewhere?

Reply:
removed